confignode/src/main/java/org/apache/iotdb/confignode/persistence/NodeInfo.java

```diff
@@ -121,7 +123,9 @@ public void addMetrics() {
               registeredDataNodes,
               Map::size,
               Tag.NAME.toString(),
-              "online");
+              "total",
+              Tag.STATUS.toString(),
+              "registered");
```

Comment:
I think the value of tag can be the name of NodeStatus enums to remove hard code.